tokio/Cargo.toml

```diff
@@ -105,8 +104,8 @@ memchr = { version = "2.2", optional = true }
 mio = { version = "0.6.20", optional = true }
 iovec = { version = "0.1.4", optional = true }
 num_cpus = { version = "1.8.0", optional = true }
-# Backs `DelayQueue`
-slab = { version = "0.4.1", optional = true }
+parking_lot = { version = "0.10.0", optional = true }
```

Comment:
We probably should add a comment saying that, including the `parking_lot` feature flag will mean adopting their MSRV as well.

Reply:
Some (all?) other features are discussed in src/lib.rs rustdoc. Would that be a good place for this? Seems like the Cargo.toml would make such a bit hard to find?

Reply:
Yes, we can add it in `lib.rs` as well.

Maybe we should just say "if you enable this feature, you get no MSRV guarantee" :)

Reply:
In bbb4356 I add `parking_lot` to src/lib documented features. It forced a clarification that `Full` is not "all features" but rather "all public API features".  Also I tried to strike a direct, fact-based tone by saying "MSRV may increase according to the _parking_lot_ release in use.".  As this is getting into Tokio project policy territory, I feel you might want to tweak this further to your liking post merge?